src/main/java/edu/cornell/kfs/module/purap/document/service/impl/IWantDocumentServiceImpl.java

```diff
@@ -558,12 +558,17 @@ private void setUpVendorSectionForReqDoc(RequisitionDocument requisitionDocument
     /**
      * Copies all attachments from the I Want document to the accounting document
      * 
+     * NOTE: A non-null extCopyNoteIndicator value will cause the copied notes to also
+     * include an extended attribute with the given copyNoteIndicator setting.
+     * 
      * @param document
      * @param iWantDocument
      * @param copyConfidentialAttachments
+     * @param extCopyNoteIndicator
      * @throws Exception
      */
-    private void copyIWantDocAttachments(AccountingDocument document, IWantDocument iWantDocument, boolean copyConfidentialAttachments) throws Exception {
+    private void copyIWantDocAttachments(AccountingDocument document, IWantDocument iWantDocument, boolean copyConfidentialAttachments,
+            Boolean extCopyNoteIndicator) throws Exception {
```

Comment:
I suppose it could be reduced to just a primitive boolean for the current use cases. I had left it as a Boolean in case we needed the ability to create extension attributes with copyNoteIndicator set to false; however, if we don't expect to be needing that at all, then I can change it to be just a primitive.